tower-web/src/app/modules/main/component/workflow-main-tabs/workflow-main-tabs.component.html

```diff
@@ -2,13 +2,13 @@
   <div class="card-header pb-0 border-bottom-0">
     <ul class="nav nav-tabs">
       <li class="nav-item">
-        <a class="nav-link active" id="commandLine-tab" data-toggle="tab" href="#commandLine" role="tab" aria-controls="commandLine">Command line</a>
+        <a class="nav-link active" id="commandLine-tab" data-toggle="tab" href="#commandLine" role="tab" aria-controls="commandLine">Command <span class="d-none d-sm-inline">line</span></a>
       </li>
       <li class="nav-item">
         <a class="nav-link" id="parameters-tab" data-toggle="tab" href="#parameters" role="tab" aria-controls="parameters">Parameters</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" id="configuration-tab" data-toggle="tab" href="#configuration" role="tab" aria-controls="configuration">Configuration</a>
+        <a class="nav-link" id="configuration-tab" data-toggle="tab" href="#configuration" role="tab" aria-controls="configuration">Config<span class="d-none d-sm-inline">uration</span></a>
       </li>
```

Comment:
wow, this is like a pro 🏆

Reply:
![image](https://user-images.githubusercontent.com/465550/69974626-eea53880-1525-11ea-86c7-71ea0fafa2c4.png)